genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/ClusterTask.java

```diff
@@ -76,8 +75,9 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
             final JobExecutionEnvironment jobExecEnv =
                 (JobExecutionEnvironment) context.get(JobConstants.JOB_EXECUTION_ENV_KEY);
             final Cluster cluster = jobExecEnv.getCluster();
-            tags.put(MetricsConstants.TagKeys.CLUSTER_NAME, cluster.getName());
-            tags.put(MetricsConstants.TagKeys.CLUSTER_ID, cluster.getId().orElse(NO_ID_FOUND));
+
+            MetricsUtils.addCommonMetricTags(jobExecEnv, tags, NO_ID_FOUND);
```

Comment:
Since `MetricsUtils` is used in many places this method name leads me to believe it is a program wide set of `commonMetricTags` when it really is just `addCommonJobWorkflowMetricTags` or something like that